pkg/shell/superuser.jsx

```diff
@@ -180,6 +187,32 @@ const UnlockDialog = ({ proxy, host }) => {
             <Button variant="secondary" className='btn-cancel' onClick={cancel}>
                 {_("Close")}
             </Button>);
+    } else if (bridges) {
+        const apply = () => {
+            start(bridge);
+        };
+
+        title = _("Switch to administrative access");
+        body = (
+            <Form isHorizontal onSubmit={event => { apply(); event.preventDefault(); return false }}>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17536-20220708-125727-77f202a1-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)